static/app/views/replays/utils/mergeBreadcrumbsEntries.tsx

```diff
@@ -0,0 +1,20 @@
+import {Entry, EntryType, Event} from 'sentry/types/event';
+
+function isBreadcrumbs(entry: Entry) {
+  return entry.type === EntryType.BREADCRUMBS;
+}
+
+export default function mergeBreadcrumbsEntries(events: Event[]) {
+  const allValues = events
+    .flatMap(event => event.entries.filter(isBreadcrumbs))
+    .flatMap(entry => entry.data.values)
+    .map(value => JSON.stringify(value));
+  const deduped = new Set(allValues);
+
+  return {
+    type: EntryType.BREADCRUMBS,
+    data: {
+      values: Array.from(deduped).map(value => JSON.parse(value)),
+    },
+  } as Entry;
```

Comment:
Is there a difference between casting the return as `Entry` rather than specifying the return type of the function?

Reply:
I don't think there's a difference internally to the function, and i'm sure call-sites won't know the difference. I can annotate the function instead though.